specs/trip_spec.rb

```diff
@@ -0,0 +1,70 @@
+require_relative 'spec_helper'
+
+describe "Trip class" do
+
+  describe "initialization of Trip" do
+    it "Initializes Trip with trip ID, rider ID, driver ID, date, and rating" do
+      trip_id = "4321"
+      driver_id = "987"
+      rider_id = "1234"
+      date = "2017-3-8"
+      rating = "5"
+
+      trip = RideShare::Trip.new(trip_id, driver_id, rider_id, date, rating)
+      trip.must_respond_to :trip_id
+      trip.trip_id.must_equal trip_id
+
+      trip.must_respond_to :date
+      trip.date.must_equal date
+
+      trip.must_respond_to :driver_id
+      trip.driver_id.must_equal driver_id
+
+      trip.must_respond_to :rider_id
+      trip.rider_id.must_equal rider_id
+
+      trip.must_respond_to :rating
+      trip.rating.must_equal rating
+      # Needs test to see that rating is between 1 and 5
+    end
+  end
+
+  describe "self.find_many_drivers class method" do
+    it "Returns array of many Trip instances for given driver" do
+      driver_trips = RideShare::Trip.find_many_drivers("1")
+      driver_trips.must_be_instance_of Array
+      driver_trips.length.must_equal 9
```

Comment:
As in driver_spec.rb, I would like to see more testing around the elements in the returned array. What type of thing did you get back? Is the data what you expected to see?